extensions/wikia/Forum/tests/ForumTest.php

```diff
@@ -34,7 +34,7 @@ function ( $pageId ) use ( $titlesFromNamespace ) {
 
 		$this->mockStaticMethodWithCallBack( 'Wall', 'newFromTitle',
 			function ( $title ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$title". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)